### PR TITLE
Add a countdown timer and dateFns library

### LINKED ISSUE #86: [FEATURE] Countdown clock
We want to display a countdown for when the next game will commence.

### PR DESCRIPTION
**Pull request by lpatmo**
## Description
This PR resolves #86. 

## Before

image

## After

image

## Implementation Details 
- [x] Used dateFns library
- [x] Put the countdown on the navbar

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -26,10 +26,15 @@ function App() {
   const [volume, setVolume] = useState(2);
   const [mobileOrSafari, setMobileOrSafari] = useState(false);
 
+  function setGameIndex() {
+    let startDate = new Date('2023-05-11');
```
Comment (arturolei) [inline src/App.jsx:30]: Why is this hard-coded?

**Review by arturolei — APPROVED** [review]
Looks good but I think `Date()` has methods which do what `date-fns` does; might be possible to avoid adding to the bundle.

File: src/App.jsx (hunk 1 above)
Comment (lpatmo) [reply src/App.jsx:30]: it reflects the launch date.

**Comment by ISPOL** [comment]
Don't like the slant on the timer, and the countdown should have more emphasis or a larger size for focus. Otherwise looks fine.

**Review by ISPOL — APPROVED** [review]

**Pull request merged**